### PR TITLE
Changes to Coalition mission descriptions

### PR DESCRIPTION
**Pull request by strollcata**
The current descriptions for Coalition missions have a lot of 'to's in them, which makes them seem somewhat unrefined, in my opinion at least. I've rewritten the descriptions in an effort to fix this. The dialogue remains unchanged.

### CODE REVIEW

File: data/coalition missions.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@
 mission "Coalition: First Contact"
 	landing
 	name "Register with the Heliarchs"
+	description "You have been informed that, should you wish to conduct business in Coalition space, you must petition the rulers of the Coalition for permission. You can pursue this course of action on the <planet>.
 	description "You have been told to travel to the <planet> to register with the rulers of Coalition space and petition them for permission to do business here."
```
Comment (jafdy) [inline data/coalition missions.txt]: Both paragraphs together don't make sense (and also might not fit).

Comment (strollcata) [inline data/coalition missions.txt]: Left the original in by mistake, should be fixed now.

File: data/coalition missions.txt [hunk 2]
```diff
@@ -11,7 +11,7 @@
 mission "Coalition: First Contact"
 	landing
 	name "Register with the Heliarchs"
-	description "You have been told to travel to the <planet> to register with the rulers of Coalition space and petition them for permission to do business here."
+	description "You have been informed that, should you wish to conduct business in Coalition space, you must petition the rulers of the Coalition for permission. You can pursue this course of action on the <planet>.
```
Comment (Bladewood) [inline data/coalition missions.txt]: closing quotation mark.

**Comment by endless-sky** [comment]
Mission descriptions generally use as concise language as possible. Adding roundabout phrases like "should you wish to conduct business" or "you can pursue this course of action" is a step in the wrong direction, in my opinion.

**Comment by strollcata** [comment]
In that case, I'll close up this pull request. I'll keep that in mind when next looking at mission descriptions.

**Pull request closed without merging**